versions.props

```diff
@@ -1,20 +1,24 @@
 ch.qos.logback:* = 1.2.3
-com.fasterxml.jackson.*:jackson-* = 2.9.7
+com.fasterxml.jackson.*:jackson-* = 2.9.8
 com.google.code.findbugs:jsr305 = 3.0.2
 com.google.errorprone:error_prone_annotations = 2.3.2
 com.google.googlejavaformat:google-java-format = 1.6
 com.google.guava:guava = 23.6.1-jre
-com.palantir.conjure.java.api:* = 2.0.0
-com.palantir.conjure.java.runtime:* = 4.3.0
+com.palantir.conjure.java.api:* = 2.2.0
+com.palantir.conjure.java.runtime:* = 4.11.1
+# dependency-upgrader:OFF
+# Regression introduced by https://github.com/palantir/conjure-java-runtime/pull/865
+com.palantir.conjure.java.runtime:conjure-java-jaxrs-client = 4.6.0
```

Comment:
Not a huge fan of running these libraries with different versions, but I think it's preferable over not upgrading.